resources/js/Pages/Collections/Components/FeaturedCollections/FeaturedCollectionsItem.tsx

```diff
@@ -0,0 +1,158 @@
+                                <NetworkIcon networkId={data.chainId} />
+                            </div>
+                        </div>
+
+                        <div className="flex flex-col ">
+                            <span className="text-sm font-medium capitalize leading-6 text-theme-secondary-700 dark:text-theme-dark-200">
+                                {t("pages.collections.featured.title")}
+                            </span>
+                            <Heading
+                                className="text-xl"
+                                level={3}
+                            >
+                                {data.name}
+                            </Heading>
+                        </div>
+                    </div>
+
+                    <div className="my-3 line-clamp-4 text-base font-medium leading-6 text-theme-secondary-700 dark:text-theme-dark-200 lg:line-clamp-2 lg:h-12">
+                        {truncateDescription(data.description)}
+                    </div>
+
+                    <div className="flex flex-col gap-6 sm:flex-row sm:items-center sm:justify-between md-lg:flex-col md-lg:items-start">
+                        <div className="flex flex-row items-center justify-start">
+                            <GridHeader
+                                className="!px-0"
+                                wrapperClassName="w-fit"
+                                title={t("common.nfts")}
+                                value={data.nftsCount}
+                            />
+                            <div className="mx-4 h-8 w-px bg-theme-secondary-300 dark:bg-theme-dark-700 sm:mx-6" />
+                            <GridHeader
+                                className="!px-0"
+                                wrapperClassName="w-fit"
+                                title={t("common.floor_price")}
+                                value={
+                                    <FormatCrypto
+                                        value={data.floorPrice ?? "0"}
+                                        token={token}
+                                    />
+                                }
+                            />
+                            <div className="mx-4 h-8 w-px bg-theme-secondary-300 dark:bg-theme-dark-700 sm:mx-6" />
+                            <GridHeader
+                                className="!px-0"
+                                wrapperClassName="w-fit"
+                                title={t("common.volume", { frequency: "" })}
+                                value={
+                                    <FormatCrypto
+                                        value={data.volume ?? "0"}
+                                        token={token}
+                                        maximumFractionDigits={2}
+                                    />
+                                }
+                            />
```

Comment:
for readability it might be nice to move this to a component, but let me know what you think